components/brave_wallet/browser/ethereum_provider_impl.h

```diff
@@ -419,7 +419,7 @@ class EthereumProviderImpl final
   EthBlockTracker eth_block_tracker_;
   EthLogsTracker eth_logs_tracker_;
   bool first_known_accounts_check_ = true;
-  PrefService* prefs_ = nullptr;
+  const raw_ptr<PrefService> prefs_ = nullptr;
```

Comment:
can we use `const raw_ref` for this? It shouldn't be null, we just never CHECK/DCHECK it

Reply:
never mind, it requires further changes when passing it. I'm fine with what it is now.

Reply:
I have a separate PR for this, dealing with these fields. I’m gonna revisit this to be a ref_ptr if that’s safe.

https://github.com/brave/brave-core/pull/18113

Reply:
Done